package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "fhir-works-on-aws-interface",
-  "version": "1.0.0",
+  "version": "2.0.0",
```

Comment:
This is a minor change? Adding a new feature not modifying existing?

Reply:
The interface package is consumed in a way that most changes are breaking changes.

Even though this PR only adds a method, it'd be a breaking change for consumers like `fhir-works-on-aws-authz-rbac` since they'll fail to compile
```
Error:(18, 14) TS2420: Class 'RBACHandler' incorrectly implements interface 'Authorization'.
  Property 'getAllowedResourceTypesForOperation' is missing in type 'RBACHandler' but required in type 'Authorization'.
```

Reply:
ops yep your right!